### PR TITLE
Gibbing Throws Internal Organs

### PR DESCRIPTION
**Pull request by Fox-McCloud**
I'm expecting this to be shot down almost instantaneously because of how it changes things.

Gibbing will now remove and throw internal organs--yes, this includes brains.

While this initially comes off as a huge paradigm shift (and it is, in part), I don't personally think it'll be as big of an impact as some might believe: yes, it makes explosions and a few other things easier to "come back from", but revival is still unlikely in a lot of cases and brains are incredibly easy to dispose of/hide/destroy; likewise, with how explosions pattern, sometimes brains end up destroyed, anyway, because gibs, the brain gets moved to a tile where it's inevitably ex_act()'d on and it's gone, forever, anyway.

Yes, this means that Lichdom wizards can revive from a brain to a new body (there's still the gibber and crematorium); I think suicide bombing an antag and wrecking a huge portion of the station in the proces is a lame+griefy tactic anyway.

:cl: Fox McCloud
rscadd: Gibbing mobs will now throw their internal organs
/:cl:

### CODE REVIEW

**Comment by neersighted** [comment]
:+1: this will make things more interesting.

**Comment by Dorsisdwarf** [comment]
Oh boy, even _less_ lethality and permadeath. Just the thing to make people fear removal from the round!

**Comment by PKPenguin321** [comment]
I was all for it until it included brains

**Comment by GunHog** [comment]
Wait, so this is not a PR to make the gibber machine itself throw brains? You have raised my hopes of having an instant braining machine as a Roboticist, and dashed them quite expertly, sir!

**Comment by bgobandit** [comment]
:-1: for brains

:+1: for everything else

**Comment by neersighted** [comment]
A gibber in robotics that throws brains is actually a great idea...

**Comment by KazeEspada** [comment]
>unironically buffing lich
You high?

**Comment by octareenroon91** [comment]
No limbs though?

**Comment by Aloraydrel** [comment]
Remove the brain part and it would be good. Gibbing is usually the intent to remove someone from the round same with cremating

File: code/modules/mob/living/carbon/carbon.dm [hunk 1]
```diff
@@ -72,6 +72,13 @@
 					src.gib()
 
 /mob/living/carbon/gib(animation = 1)
+	death(1)
+	for(var/obj/item/organ/internal/I in internal_organs)
+		if(I)
+			I.Remove(src)
+			I.loc = get_turf(src)
```
Comment (neersighted) [inline code/modules/mob/living/carbon/carbon.dm]: `forceMove`

**Comment by Niknakflak** [comment]
:-1: brains
:+1: everything else

**Comment by octareenroon91** [comment]
Add brain drop as an optional parameter to gib.

**Comment by phil235** [comment]
I'm with octareenroon91, give an argument to gib() so we can choose to drop all organs, only the brain, or none.

**Comment by Incoming5643** [comment]
Realistically here liches shouldn't even HAVE (nonlimb) organs to throw

**Comment by KorPhaeron** [comment]
@Fox-McCloud 

Do you plan on updating this to allow arguments on which if any organs get thrown?

**Comment by Fox-McCloud** [comment]
@KorPhaeron 

I can; just been busy, though I personally think that brains should be thrown, by default---this is the default behavior for cyborgs (MMI is ejected, and those can even talk), and as detailed elsewhere, MMI's aren't recovered that frequently...don't think it would be any different with brains. Not to mention that with @phil235's changes, it's easier than ever to damage a brain.

I can still add it though, if absolutely necessary.

**Comment by Fox-McCloud** [comment]
@KorPhaeron 

Updated with the argument to not drop the brain; default to off.

**Comment by Cheridan** [comment]
ok then

**Comment by phil235** [comment]
This fixes #15263 and #13797.

**Pull request merged**